source/app/service-providers/fsal/util/safe-delete.ts

```diff
@@ -7,7 +7,7 @@ export async function safeDelete (absPath: string, deleteOnFail: boolean): Promi
   } catch (err: any) {
     if (deleteOnFail) {
       // If this function throws, there's really something off and we shouldn't recover.
-      await fs.unlink(absPath)
+      await fs.rm(absPath, { recursive: true, force: true })
```

Comment:
It might make sense to add a log entry here, because `force: true` will silently swallow errors

Reply:
I added an error log message.